common-lib/src/main/scala/com/gu/mediaservice/lib/argo/model/EmbeddedEntity.scala

```diff
@@ -19,7 +20,8 @@ object EmbeddedEntity extends WriteHelpers {
   implicit def embeddedEntityWrites[T: Writes]: Writes[EmbeddedEntity[T]] = (
     (__ \ "uri").write[String].contramap((_: URI).toString) ~
       (__ \ "data").writeNullable[T] ~
-      (__ \ "links").writeNullable[List[Link]].contramap(someListOrNone[Link])
+      (__ \ "links").writeNullable[List[Link]].contramap(someListOrNone[Link]) ~
+      (__ \ "actions").writeNullable[List[Action]].contramap(someListOrNone[Action])
```

Comment:
Thinking about this further - I'm assuming it'll be promised based?
e.g.
```JavaScript
resource.getAction('delete').then(action => { /* show action content */ })
```
Either way this is probably a discussion for the theseus client.

I suppose the question here is whether to have an empty array to say there are no actions, or no array.